pkgs/servers/etebase/default.nix

```diff
@@ -55,7 +55,7 @@ python.pkgs.buildPythonPackage rec {
   '';
 
   passthru.updateScript = nix-update-script {};
-  passthru.python = python;
+  passthru.python = python';
   # PYTHONPATH of all dependencies used by the package
   passthru.pythonPath = python.pkgs.makePythonPath propagatedBuildInputs;
```

Comment:
```suggestion
  passthru.pythonPath = python'.pkgs.makePythonPath propagatedBuildInputs;
```

Reply:
Thanks! Not sure how i missed that with search & replace.